satpy/etc/areas.yaml

```diff
@@ -1,3 +1,212 @@
+
+
+# ------------------------------ Meteosat Second Generation (MSG) / SEVIRI Instrument ----------------------------------
+
+# Full disk
+
+msg_seviri_fes_3km:
+  description: MSG SEVIRI Full Earth Scanning service area definition with 3 km resolution
+  projection:
+    proj: geos
+    lon_0: 0.0
+    a: 6378169.0
+    b: 6356583.8
+    h: 35785831.0
+  shape:
+    height: 3712
+    width: 3712
+  area_extent:
+    lower_left_xy: [-5570248.686685662, -5567248.28340708]
+    upper_right_xy: [5567248.28340708,   5570248.686685662]
+
+msg_seviri_fes_1km:
+  description: MSG SEVIRI Full Earth Scanning service area definition with 1 km resolution
+  projection:
+    proj: geos
+    lon_0: 0.0
+    a: 6378169.0
+    b: 6356583.8
+    h: 35785831.0
+  shape:
+    height: 11136
+    width: 11136
+  area_extent:
+    lower_left_xy: [-5571248.412732527, -5566247.740968115]
+    upper_right_xy: [5566247.740968115,  5571248.412732527]
+
+msg_seviri_rss_3km:
+  description: MSG SEVIRI Rapid Scanning Service area definition with 3 km resolution
+  projection:
+    proj: geos
+    lon_0: 9.5
+    a: 6378169.0
+    b: 6356583.8
+    h: 35785831.0
+  shape:
+    height: 3712
+    width: 3712
+  area_extent:
+    lower_left_xy: [-5570248.686685662, -5567248.28340708]
+    upper_right_xy: [5567248.28340708,   5570248.686685662]
+
+msg_seviri_rss_1km:
+  description: MSG SEVIRI Rapid Scanning Service area definition with 1 km resolution
+  projection:
+    proj: geos
+    lon_0: 9.5
+    a: 6378169.0
+    b: 6356583.8
+    h: 35785831.0
+  shape:
+    height: 11136
+    width: 11136
+  area_extent:
+    lower_left_xy: [-5571248.412732527, -5566247.740968115]
+    upper_right_xy: [5566247.740968115,  5571248.412732527]
+
+msg_seviri_iodc_3km:
+  description: MSG SEVIRI Indian Ocean Data Coverage service area definition with 3 km resolution
+  projection:
+    proj: geos
+    lon_0: 41.5
+    a: 6378169.0
+    b: 6356583.8
+    h: 35785831.0
+  shape:
+    height: 3712
+    width: 3712
+  area_extent:
+    lower_left_xy: [-5570248.686685662, -5567248.28340708]
+    upper_right_xy: [5567248.28340708,   5570248.686685662]
+
+msg_seviri_iodc_1km:
+  description: MSG SEVIRI Indian Ocean Data Coverage service area definition with 1 km resolution
+  projection:
+    proj: geos
+    lon_0: 41.5
+    a: 6378169.0
+    b: 6356583.8
+    h: 35785831.0
+  shape:
+    height: 11136
+    width: 11136
+  area_extent:
+    lower_left_xy: [-5571248.412732527, -5566247.740968115]
+    upper_right_xy: [5566247.740968115,  5571248.412732527]
+
+# Regional
+
+EuropeCanary:
+  description: Northern disk MSG image 0 degrees
+  projection:
+    proj: geos
+    lon_0: 0.0
+    a: 6378169.0
+    b: 6356583.8
+    h: 35785831.0
+  shape:
+    height: 1200
+    width: 3000
+  area_extent:
+    lower_left_xy: [-4823148.089050828, 1969764.6783588605]
+    upper_right_xy: [4178061.408400173, 5570248.477339261]
+
+EastEurope:
```

Comment:
Is this an area that existed before? If so, did it have the same name? Regardless, EastEurope, EuropeCanary, and SouthAmerica are pretty generic names for geostationary specific areas. Obviously, whatever amount of South America SEVIRI sees in its geostationary full disk isn't going to be the best for other datasets that are more "central" to South America.

Reply:
Ah I see now that these were old areas. Did we decide we'd come up with better names for these in a later PR?

Reply:
Yeah, those are all old areas. And yes, I think another PR is needed for the general cleanup/renaming, and possibly adding a deprecation warning. For now, I relegated to deprecation only the old full-disk ones.